_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -123,27 +124,60 @@ module "forseti-on-gke-with-config-validator" {
     # SSH Git repository location, usually in the following
     # format: git@repo-host:repo-owner/repo-name.git
     policy_library_repository_url = ""
+
+    # Whether or not to enable the periodic git-sync.
+    policy_library_sync_enabled   = true
 }
 ```
+#### Policy Library in a GCS bucket
+```bash
+module "forseti-on-gke-with-config-validator" {
+    # Other parameters/variables removed for brevity
 
-#### With Helm
+    # Enable config-validator
+    config_validator_enabled = true
+}
+```
 
+### With Helm
+#### Policy Library in a Git Repository
 In the Helm example above, the following variables are required in the user defined *values.yaml* file.
 
 ```yaml
-# configValidator sets whether or not to deploy config-validator
-configValidator: true
+# configValidator.enabled sets whether or not to deploy config-validator
+configValidator.enabled: true
 
-# gitSyncPrivateSSHKey is the private OpenSSH key generated to allow the git-sync to clone the policy library repository.
-gitSyncPrivateSSHKey: ""
+# configValidator.policyLibrary.gitSync.privateSSHKey
+# is the private OpenSSH key generated to allow the 
+# git-sync to clone the policy library repository.
+configValidator.policyLibrary.gitSync.privateSSHKey: ""
 
-# gitSyncSSH use SSH for git-sync operations
-gitSyncSSH: true
+# configValidator.policyLibrary.repositoryURL is a git
+# repository policy-library.
+configValidator.policyLibrary.repositoryURL: ""
 
-# policyLibraryRepositoryURL is a git repository policy-library.
-policyLibraryRepositoryURL: ""
 
+# configValidator.policyLibrary.gitSync.enabled 
+# determines if the Git repository will be polled
+# periodically for changes.
+configValidator.policyLibrary.gitSync.enabled: true
 ```
+#### Policy Library in a GCS bucket
+In the Helm example above, the following variables are required in the user defined *values.yaml* file.
+
+```yaml
+# configValidator.policyLibrary.bucket is the GCS
+# storage bucket containing the policy-library.
+# This overrides policyLibrary.respositoryURL.
+# Ommit the gs://.
```

Comment:
Typo, you really like this word `ommit` :)

Reply:
ack